pipelinewise/fastsync/s3_csv_to_snowflake.py

```diff
@@ -18,8 +18,6 @@
 
 REQUIRED_CONFIG_KEYS = {
     'tap': [
-        'aws_access_key_id',
```

Comment:
@avinash-ankur are there downstream changes in the tap we need to make, for wherever these are being consumed? like check for them and not insert them into the client string if they don't exist?